src/ert/scheduler/lsf_driver.py

```diff
@@ -440,6 +452,14 @@ async def poll(self) -> None:
                     f"bjobs gave returncode {process.returncode} and error {stderr.decode()}"
                 )
             bjobs_states = _parse_jobs_dict(parse_bjobs(stdout.decode(errors="ignore")))
+            bjobs_exec_hosts = parse_bjobs_exec_hosts(stdout.decode(errors="ignore"))
+
+            for jobid, exec_hosts in bjobs_exec_hosts.items():
+                if self._jobs[jobid].exec_hosts == "-":
```

Comment:
This condition is never true in the tests.

Reply:
I rewrote this to get better coverage. Also, this is the default value for `exec_hosts`, and the value `bjobs` will yield when no node is yet assigned for the job.
A re-run of codecov would have been nice.